docs/standard/library-guidance/cross-platform-targeting.md

```diff
@@ -107,17 +107,17 @@ public static class GpsLocation
 </Project>
 ```
 
-**✔️ CONSIDER** using [MSBuild.Sdk.Extras](https://github.com/onovotny/MSBuildSdkExtras) when multi-targeting for UWP and Xamarin as it greatly simplifies your project file.
+**![yes icon](../../media/yes.png) CONSIDER** using [MSBuild.Sdk.Extras](https://github.com/onovotny/MSBuildSdkExtras) when multi-targeting for UWP and Xamarin as it greatly simplifies your project file.
```

Comment:
I know this is still draft, but just a small comment.
I see there are 3 words being used in this article: **DO**, **AVOID**, and **CONSIDER**.

I suspect that DO means that you should do that, while CONSIDER means that you **may** do that ?

Reply:
Thanks for pointing this out. DO and CONSIDER used the same original checkmark character, so I think my change is okay. Perhaps I should change the alt text to "checkmark icon" though.